reana_job_controller/kubernetes_job_manager.py

```diff
@@ -12,17 +10,17 @@
 import logging
 import os
 import traceback
-import uuid
+from typing import Optional
 
 from flask import current_app
 from kubernetes import client
 from kubernetes.client.models.v1_delete_options import V1DeleteOptions
 from kubernetes.client.rest import ApiException
+
```

Comment:
No need to add an empty line here. Both `kubernetes` and `reana-commons` are third-party libraries from the Job Controller point of view.